.github/workflows/pr-checks.yml

```diff
@@ -10,9 +10,95 @@ permissions:
   contents: read
   packages: write
```

Comment:
Can we change this to `package:read`? I don't think anything in this workflow needs write permissions